src/command_modules/azure-cli-acr/azure/cli/command_modules/acr/task.py

```diff
@@ -53,34 +60,62 @@ def acr_task_create(cmd,  # pylint: disable=too-many-locals
                     resource_group_name=None,
                     target=None,
                     auth_mode=None):
+
+    registry, resource_group_name = get_registry_by_name(
+        cmd.cli_ctx, registry_name, resource_group_name)
+
+    if context_path.lower() == NULL_CONTEXT:
+        context_path = None
+        commit_trigger_enabled = False
+        pull_request_trigger_enabled = False
+
     if (commit_trigger_enabled or pull_request_trigger_enabled) and not git_access_token:
         raise CLIError("If source control trigger is enabled [--commit-trigger-enabled] or "
                        "[--pull-request-trigger-enabled] --git-access-token must be provided.")
 
-    if file.endswith(ALLOWED_TASK_FILE_TYPES):
-        FileTaskStep = cmd.get_models('FileTaskStep')
-        step = FileTaskStep(
-            task_file_path=file,
-            values_file_path=values,
+    if cmd_value:
+        yaml_template = "steps: \n  - cmd: {{ .Values.image }}\n    timeout: {{ .Values.timeout }}\n"
```

Comment:
nit: I would rather name the values for `cmd` as `command` instead of `image` as it can more than just an image and confused me.
 `- cmd: {{ .Values.command}}`